docs/management/connectors/action-types/opsgenie.asciidoc

```diff
@@ -0,0 +1,173 @@
+
+[float]
+[[opgenie-connector-networking-configuration]]
+==== Connector networking configuration
+
+Use the <<action-settings, Action configuration settings>> to customize connector networking configurations, such as proxies, certificates, or TLS settings. You can set configurations that apply to all your connectors or use `xpack.actions.customHostSettings` to set per-host configurations.
+
+[float]
+[[Preconfigured-opsgenie-configuration]]
+==== Preconfigured connector type
+
+[source,text]
+--
+ my-opsgenie:
+   name: preconfigured-opsgenie-connector-type
+   actionTypeId: .opsgenie
+   config:
+    apiUrl: https://api.opsgenie.com
+   secrets:
+     apiKey: apikey
+--
+
+Config defines information for the connector type.
+
+`apiUrl`:: A string that corresponds to *URL*.
+
+Secrets defines sensitive information for the connector type.
+
+`apiKey`:: A string that corresponds to *API Key*.
+
+[float]
+[[define-opsgenie-ui]]
+==== Define connector in Stack Management
+
+Define Opsgenie connector properties.
+
+[role="screenshot"]
+image::management/connectors/images/opsgenie-connector.png[Opsgenie connector]
+
+Test Opsgenie action parameters.
+
+[role="screenshot"]
+image::management/connectors/images/opsgenie-params-test.png[Opsgenie params test]
+
+[float]
+[[opsgenie-action-configuration]]
+==== Action configuration
+
+The Opsgenie connector supports two types of actions: Create Alert and Close Alert. The properties supported for each action are different because Opsgenie defines different properties for each operation.
+
+When testing the Opsgenie connector, choose the appropriate action from the selector. Each action has different properties that can be configured.
+
+Action::    Whether to create or close an alert within Opsgenie. When using the Opsgenie connector for a rule action, the Action selector is hidden and automatically set to the appropriate value. If the action is run when a rule fires, it will create an alert within Opsgenie. If the action is run when a rule recovers, it will close an alert within Opsgenie.
+
+[float]
+[[opsgenie-action-create-alert-configuration]]
+===== Configure the create alert action
+
+You can configure the create alert action through the form view or using a JSON editor.
+
+[float]
+[[opsgenie-action-create-alert-form-configuration]]
+====== Form configuration
+
+The create alert action form has the following configuration properties.
+
+Message::   The message for the alert. This field is required.
+Opsgenie Tags::   The tags for the alert.
+Priority::  The priority level for the alert.
+Description::   The description for the alert to provide detailed information about the alert.
+Alias::   The identifier for the alert. The `Alias` is used for https://support.atlassian.com/opsgenie/docs/what-is-alert-de-duplication/[alert de-duplication].
```

Comment:
This is another case where I think we should be clearer we're sending them outside our docs:
```suggestion
Alias::   The alert identifier, which is used for alert de-duplication in Opsgenie. For more information, refer to the https://support.atlassian.com/opsgenie/docs/what-is-alert-de-duplication/[Opsgenie documentation].
```